src/components/related-assets/types.ts

```diff
@@ -0,0 +1,5 @@
+export interface RelatedAsset {
```

Comment:
Given we know it's only one type and that it's unlikely for more types to come, I'd have left this as an interface in the original file